### PR TITLE
Use new dconf bash functions for bash scripts and add some missing dconf scripts

### PR DESCRIPTION
**Pull request by redhatrises**
#### Description:

- Use new dconf bash functions for bash scripts and add some missing dconf scripts

### CODE REVIEW

File: shared/fixes/bash/dconf_gnome_banner_enabled.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# platform = Red Hat Enterprise Linux 7
+. /usr/share/scap-security-guide/remediation_functions
+
+dconf_settings 'org/gnome/login-screen' 'banner-message-enable' "true" 'gdm.d' '00-security-settings'
```
Comment (shawndwells) [inline shared/fixes/bash/dconf_gnome_banner_enabled.sh]: double vs single quote on ``"true"``

Comment (redhatrises) [reply shared/fixes/bash/dconf_gnome_banner_enabled.sh]: @shawndwells nitpicker ;)

Fixed your nitpicking.

**Comment by shawndwells** [comment]
just waiting on jenkins

File: shared/fixes/bash/dconf_gnome_banner_enabled.sh (hunk 1 above)
Comment (shawndwells) [reply shared/fixes/bash/dconf_gnome_banner_enabled.sh]: "nickpicking'

Comment (redhatrises) [reply shared/fixes/bash/dconf_gnome_banner_enabled.sh]: hahaha

**Pull request merged**